tests/integration_tests/albumentations_test.py

```diff
@@ -161,6 +166,202 @@ def test_coco_segmentation_albumentations_intergration(self):
+            [255, 127, 14],  # Left Hip -> Left Knee
+            [214, 39, 40],  # Right Hip -> Right Knee
+            [31, 119, 180],  # Left Knee -> Left Ankle
+            [44, 160, 44],  # Right Knee -> Right Ankle
+        ]
+
+        keypoint_colors = [
+            [148, 103, 189],
+            [31, 119, 180],
+            [148, 103, 189],
+            [31, 119, 180],
+            [148, 103, 189],
+            [31, 119, 180],
+            [148, 103, 189],
+            [31, 119, 180],
+            [148, 103, 189],
+            [31, 119, 180],
+            [148, 103, 189],
+            [31, 119, 180],
+            [148, 103, 189],
+            [31, 119, 180],
+            [148, 103, 189],
+            [31, 119, 180],
+            [148, 103, 189],
+        ]
+
+        from super_gradients.training.transforms import KeypointsRescale
+
+        transforms = [
+            KeypointsRescale(height=320, width=640),
+            {
+                "Albumentations": {
+                    "Compose": {
+                        "transforms": [{"RandomBrightnessContrast": {"p": 1}}, {"RandomCrop": dict(width=300, height=320)}],
+                        "bbox_params": {
+                            "min_area": 1,
+                            "min_visibility": 0,
+                            "min_width": 0,
+                            "min_height": 0,
+                            "check_each_transform": True,
+                        },
+                        "keypoint_params": {
+                            "check_each_transform": True,
+                        },
+                    },
+                }
+            },
+        ]
+
+        ds = COCOPoseEstimationDataset(
+            data_dir=mini_coco_data_dir,
+            images_dir="images/val2017",
+            json_file="annotations/person_keypoints_val2017.json",
+            include_empty_samples=True,
+            edge_links=edge_links,
+            edge_colors=edge_colors,
+            keypoint_colors=keypoint_colors,
+            transforms=transforms,
+        )
+
+        sample = next(iter(ds))
+
+        bboxes_xyxy = xywh_to_xyxy(bboxes=np.array(sample.bboxes_xywh), image_shape=sample.image.shape)
+
+        image_with_keypoints = PoseVisualization.draw_poses(
+            image=np.array(sample.image),
+            poses=sample.joints,
+            boxes=bboxes_xyxy,
+            scores=None,
+            is_crowd=sample.is_crowd,
+            edge_links=edge_links,
+            edge_colors=edge_colors,
+            keypoint_colors=keypoint_colors,
+            show_keypoint_confidence=False,
+            joint_thickness=None,
+            box_thickness=None,
+            keypoint_radius=None,
+        )
+
+        visualize_image(image=image_with_keypoints)
+
+        # Make sure we raise an error when using unsupported transforms
+        with self.assertRaises(TypeError):
+            transforms = [
+                KeypointsRescale(height=320, width=640),
```

Comment:
Additional test case: It would be great if we could add a native transform after the albumentations part.

Reply:
Like this c9a0b749275e4bcf0f491697a1075f8cd82b6cb8 ?